### ISSUE TITLE
Validation issues

### ISSUE THREAD

**Issue by andythenorth** [OpenTTD/website#30]
1. WAVE toolbar reports 39 contrast errors for the home page, these are primarily in the layout templates, and will be present on most pages.

2. WAVE toolbar reports accessibility failures on the download pages, primarily form labels missing.

**Comment by andythenorth** [comment]
I am planning to fix these, but not a priority for me.  

If anyone else wants to resolve them:

- WAVE is a free validation tool from WebAIM https://wave.webaim.org/
- I use it via the Chrome plugin
- I run the OpenTTD website locally using _jekyll serve_, then check one page at a time
- the contrast checker sometimes gives spurious failures, as it can't always resolve the css hierarchy to work out what the user will actually see rendered, but it's generally a pretty good guide, and I tend to fix the CSS so all the errors are resolved, it's usually not much work.

**Comment by TrueBrain** [comment]
Seems the priority is this low, that it isn't being fixed ;) Going to close the issue now. If anyone wants to send in PRs, they are more than welcome!
